.github/workflows/validate.yml

```diff
@@ -16,19 +16,19 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.13, 12, 14, 15]
+        node: [10.13, 10, 12, 14, 15]
```

Comment:
Let's just remove node 10 completely. It reaches end of life on the 30th of April anyway.